src/components/ProfileDropdown.jsx

```diff
@@ -66,66 +65,79 @@ const useStyles = makeStyles((theme) => ({
 export default function ProfileDropdown(props) {
   const classes = useStyles();
   const { signOut, name, email, photo } = props;
+
   const [dropMenu, setDropMenu] = React.useState(null);
+  const handleClick = (event) => setDropMenu(event.currentTarget);
+  const handleClose = () => setDropMenu(null);
 
-  const handleClick = (event) => {
-    setDropMenu(event.currentTarget);
-  };
+  // needs to be updated once there is a working manage profile page
+  const manageProfileLink = '/home';
+
+  function createMenuItem(text, icon, task) {
+    return (
+      <>
```

Comment:
These aren't needed when there's only one child.

Reply:
OK right, im not sure why it worked now as when I originally removed them there was compiler issues. Its removed now tho and no compiler errors